### ISSUE TITLE
[ISSUE] Empty Embed Field for Jisho

### ISSUE THREAD

**Issue by No767** [No767/Rin#94]
### Current Behavior

When searching up some words, it returns with a 400, due to an empty field. A good example is haku (履く), where one of the fields is empty.

### Expected Behavior

Return some type of character, or return the field that is needed

### Steps To Reproduce

1. Search using the .jisho cmd
2. Insert "haku" for the search arg
3. Wait for it to return a 400 due to an empty field

### Environment

```markdown
- OS: Android 11/Windows 10 Pro
- Python Version: 3.10.0
- pip version: 21.3.1
- pipenv version: 2021.11.5post0

Happens on discord only
```

### Anything else?

![Screenshot_20211108-105444-904](https://user-images.githubusercontent.com/73260931/140800964-1c92ad8d-91e1-40ee-9f73-a76e47e0b670.png)

This happens on discord. This bug is on the production version for Rin

### Is there an existing issue for this?

- [X] I have searched the existing issues

**Comment by No767** [comment]
the other way to solve this is to turn it into a list and append the brackets at the end

**Comment by No767** [comment]
Fixed within commit e8e0ba0
